test/cmocka/src/audio/component/comp_set_state.c

```diff
@@ -54,7 +54,8 @@ struct test_case {
 
 #define TEST_CASE(type, in_state, cmd, out_state) \
 	{(type), (in_state), (cmd), (out_state), \
-	("comp_set_state__" #type "__" #in_state "__" #cmd "__" #out_state)}
+	("test_audio_comp_set_state__" \
```

Comment:
Can we have `test_audio_component_comp_set_state` instead of `test_audio_comp_set_state` because it's for `comp_set_state` function in header component.h ?

Reply:
Thank you sir ;)